### ISSUE TITLE
SE: Version control for swell_static_files

### ISSUE THREAD

**Issue by ashiklom** [GEOS-ESM/swell#387]
From @Dooruk on #360 (https://github.com/geos-esm/swell/issues/360#issuecomment-2210878535)

> This is a separate concern but during the SWELL development meeting there was a brief discussion on **version control with the swell_static_files**. Sounds like there is no straightforward solution to that since it requires handling many different large file formats. My suggestion for now is that we can name the static files as the SWELL version and keep the previous static folders up to 4-5 most recent versions.

**Comment by Dooruk** [comment]
Is this an active issue?

@mer-a-o and @jeromebarre are trying to add files to `advda` for `geos_cf`. We could also revisit this problem.

**Comment by Dooruk** [comment]
Also relevant one:

https://github.com/GEOS-ESM/swell/issues/586

**Comment by mranst** [comment]
I see the suggestion about having directories named after the Swell version, would this just be as simple as having separate versions in directories that are changed by swell PRs (kind of like how we go about updating the JEDI build in Swell)? I also see some discussion about using `git-lfs` on other issues, though this seems to raise other concerns. From what I can tell, the current static files take up 172G

**Comment by Dooruk** [comment]
> I see the suggestion about having directories named after the Swell version, would this just be as simple as having separate versions in directories that are changed by swell PRs (kind of like how we go about updating the JEDI build in Swell)? I also see some discussion about using `git-lfs` on other issues, though this seems to raise other concerns. From what I can tell, the current static files take up 172G

Yeah, the bottomline is there is no free version of version control for large files AFAIK. https://git-annex.branchable.com/ was brought up two years ago but I don't know how useful is that.

Bulk of that 172G filesize come from `/geos` side (i need to clean up) and most of those don't need to be version controlled. You can ignore the `geos/restarts` and `geos/run_dirs` parts. With the new GEOS approach I introduced I am using only links for `homdirs` and `expdirs`.

**Comment by mer-a-o** [comment]
I still don't have edit access to `advda`.
